### PR TITLE
Added dark mode and bug fixes

### PR DESCRIPTION
**Pull request by Bryanroal**
Was resolved a problem with the button of login with google problem and was accommodate the part of links in the form.

### CODE REVIEW

File: app/assets/stylesheets/components/_buttons.scss [hunk 1]
```diff
@@ -10,10 +9,12 @@
     margin: 0 10px 20px 30px;
     outline: none;
     width: 80%;
+    @include themify {
```
Comment (juperameza) [inline app/assets/stylesheets/components/_buttons.scss]: Is this the best way to do it??
It seems youre repeating the code too much

**Review by juperameza — CHANGES_REQUESTED** [review]
Same for the others themify 
Also dont do commits on past tense 
Add not added

File: app/assets/stylesheets/components/_buttons.scss (hunk 1 above)
Comment (juperameza) [reply app/assets/stylesheets/components/_buttons.scss]: And update the branch. that button shouldn't be there you should constantly be doing pulls from main

Comment (Bryanroal) [reply app/assets/stylesheets/components/_buttons.scss]: Done

File: app/views/devise/sessions/new.html.erb [hunk 2]
```diff
@@ -29,4 +25,4 @@
   </div>
 <% end %>
     <%= render "devise/shared/links" %>
-</div>
+</div>
```
Comment (juperameza) [inline app/views/devise/sessions/new.html.erb]: add a new line at the end of the file

**Review by juperameza — CHANGES_REQUESTED** [review]

File: app/views/devise/sessions/new.html.erb (hunk 2 above)
Comment (Bryanroal) [reply app/views/devise/sessions/new.html.erb]: done

**Review by juperameza — APPROVED** [review]

**Review by MauricioNC — APPROVED** [review]

**Review by BlazeDex — APPROVED** [review]

**Pull request merged**